cmd/k8s-operator/e2e/main_test.go

```diff
@@ -6,167 +6,120 @@ package e2e
 import (
 	"context"
 	"errors"
-	"fmt"
 	"log"
 	"os"
-	"slices"
 	"strings"
 	"testing"
+	"time"
 
-	"github.com/go-logr/zapr"
-	"github.com/tailscale/hujson"
-	"go.uber.org/zap/zapcore"
 	"golang.org/x/oauth2/clientcredentials"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"sigs.k8s.io/controller-runtime/pkg/client"
-	logf "sigs.k8s.io/controller-runtime/pkg/log"
-	kzap "sigs.k8s.io/controller-runtime/pkg/log/zap"
 	"tailscale.com/internal/client/tailscale"
+	"tailscale.com/ipn/store/mem"
+	"tailscale.com/tsnet"
 )
 
-const (
-	e2eManagedComment = "// This is managed by the k8s-operator e2e tests"
-)
+// Required ACLs:
+// {
+// 	"tagOwners": {
```

Comment:
It would be nifty to have this somewhere in an actual JSON file to make copy and paste easy